gdk4-wayland/Cargo.toml

```diff
@@ -17,6 +17,7 @@ version.workspace = true
 v4_4 = ["ffi/v4_4", "gdk/v4_4"]
 v4_10 = ["ffi/v4_10", "v4_4", "gdk/v4_10"]
 v4_12 = ["ffi/v4_12", "v4_10", "gdk/v4_12"]
+v4_16 = ["ffi/v4_16", "v4_12", "gdk/v4_16"]
```

Comment:
You probably want to edit the sys `Cargo.toml`s to only require 4.15 for the time being (and maybe update the 4.14 ones from 4.13 to 4.14?)

Reply:
that is already done in https://github.com/gtk-rs/gtk4-rs/pull/1717/commits/b4ed2863ae0ab03dd8bdc6928473d1c28085e139